app/components/MovieElement.js

```diff
@@ -1,16 +1,24 @@
-var React = require('react');
+import React from 'react';
+import PropTypes from 'react-proptypes';
 
-class MovieElement extends React.Component {
-  // Title, Image, year
-  render() {
-    return (
-      <div>
-        <img src={this.props.imageSource} />
-        <div>{this.props.title}</div>
-        <div>{this.props.year}</div>
-      </div>
-    )
-  }
+function MovieElement(props) {
+  const { imageSource } = props;
+  const { title } = props;
+  const { year } = props;
```

Comment:
You should get these values in the same line:
`const { imageSource, title, year } = props;`

Reply:
In fact, you can replace `props` to `{ imageSource, title, year }`

Now:
`function MovieElement(props) {`
after:
`function MovieElement({ imageSource, title, year }) {`